lib/main.dart

```diff
@@ -1,113 +1,185 @@
-      // _counter without calling setState(), then the build method would not be
-      // called again, and so nothing would appear to happen.
-      _counter++;
-    });
-  }
+  final double blurRadius;
+  final double spreadRadius;
+  final double circleAvatarRadius;
+  final double titleFontSize;
+  final double subtitleFontSize;
+  final double letterSpacing;
+  final double dividerHeight;
+  final double cardSubtitleFontSize;
 
   @override
   Widget build(BuildContext context) {
-    // This method is rerun every time setState is called, for instance as done
-    // by the _incrementCounter method above.
-    //
-    // The Flutter framework has been optimized to make rerunning build methods
-    // fast, so that you can just rebuild anything that needs updating rather
-    // than having to individually change instances of widgets.
-    return Scaffold(
-      appBar: AppBar(
-        // Here we take the value from the MyHomePage object that was created by
-        // the App.build method, and use it to set our appbar title.
-        title: Text(widget.title),
-      ),
-      body: Center(
-        // Center is a layout widget. It takes a single child and positions it
-        // in the middle of the parent.
-        child: Column(
-          // Column is also a layout widget. It takes a list of children and
-          // arranges them vertically. By default, it sizes itself to fit its
-          // children horizontally, and tries to be as tall as its parent.
-          //
-          // Invoke "debug painting" (press "p" in the console, choose the
-          // "Toggle Debug Paint" action from the Flutter Inspector in Android
-          // Studio, or the "Toggle Debug Paint" command in Visual Studio Code)
-          // to see the wireframe for each widget.
-          //
-          // Column has various properties to control how it sizes itself and
-          // how it positions its children. Here we use mainAxisAlignment to
-          // center the children vertically; the main axis here is the vertical
-          // axis because Columns are vertical (the cross axis would be
-          // horizontal).
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            Text(
-              'You have pushed the button this many times:',
-            ),
-            Text(
-              '$_counter',
-              style: Theme.of(context).textTheme.headline4,
+    return MaterialApp(
+      debugShowCheckedModeBanner: false,
+      title: 'Business Card App',
+      home: Scaffold(
+        body: Container(
+          decoration: BoxDecoration(
+            gradient: LinearGradient(colors: [
+              Colors.green,
+              Colors.yellow,
+            ]),
+          ),
+          child: SafeArea(
+            child: Padding(
+              padding: const EdgeInsets.only(
+                left: 50.0,
+                right: 50.0,
```

Comment:
You can use a different EdgeInsets configuration, called "symmetric"

```suggestion
              padding: const EdgeInsets.symmetric(
                horizontal: horizontalPadding
```

"horizontalPadding" should be your new constant for that value